src/plugins/vis_types/vega/public/vega_inspector/vega_adapter.ts

```diff
@@ -159,4 +160,16 @@ export class VegaAdapter {
       map((debugValues) => JSON.stringify(debugValues.spec, null, 2))
     );
   }
+
+  getErrorSubscription() {
```

Comment:
nit: what do you think about renaming it to`getErrorObservable`. It also ugly, but `subscription` means what is returned from `subscribe`